htdocs/projet/class/project.class.php

```diff
@@ -633,9 +633,7 @@ function delete($user, $notrigger=0)
 		$this->getLinesArray($user);
 
 		// Delete tasks
-		foreach($this->lines as &$task) {
-			$task->delete($user);
-		}
+		$this->deleteTasks($this->lines);
```

Comment:
Please provide the $user as parameter because it is received as parameter in delete() so we should not change behaviour by getting it as a global var later.

Reply:
I fixed it by passing the user instead of an array